.github/workflows/wheels.yml

```diff
@@ -178,39 +178,45 @@ jobs:
           if [[ ${{ matrix.buildplat[0] }} == 'macos-14' && ${{ matrix.buildplat[4] }} == '12.3' ]]; then
             CIBW+=" && echo \$(ls {dest_dir}) && \
             mv {dest_dir}/*.whl \$(find {dest_dir} -type f -name '*.whl' | sed 's/12_3/12_0/')"
-          fi          
+          fi
           echo "CIBW_REPAIR_WHEEL_COMMAND_MACOS=$CIBW" >> "$GITHUB_ENV"
 
       - name: Inject environment variable for python dev version
-        if: matrix.python[1] == '3.13'
+        if: ${{ matrix.python[1] == '3.13' && !endsWith(matrix.python[0], 't')}}
         shell: bash
         run: |
           # For dev versions of python need to use wheels from scientific-python-nightly-wheels
           # When the python version is released please comment out this section, but do not remove
           # (there will soon be another dev version to target).
           DEPS0="pip install --pre -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy"
           DEPS1="pip install ninja meson-python pybind11 pythran cython"
-          
+
           CIBW="$DEPS0;$DEPS1;bash {project}/tools/wheels/cibw_before_build_linux.sh {project}"
           echo "CIBW_BEFORE_BUILD_LINUX=$CIBW" >> "$GITHUB_ENV"
-          
+
           CIBW="$DEPS0 && $DEPS1 && bash {project}/tools/wheels/cibw_before_build_win.sh {project}"
           echo "CIBW_BEFORE_BUILD_WINDOWS=$CIBW" >> "$GITHUB_ENV"
-          
+
           CIBW="$DEPS0;$DEPS1;bash {project}/tools/wheels/cibw_before_build_macos.sh {project}"
           echo "CIBW_BEFORE_BUILD_MACOS=$CIBW" >> "$GITHUB_ENV"
 
           echo "CIBW_BEFORE_TEST=$DEPS0" >> "$GITHUB_ENV"
-          
+
+      - name: Disable build isolation for python dev version
+        if: matrix.python[1] == '3.13'
+        shell: bash
+        run: |
           CIBW="pip; args: --no-build-isolation"
-          echo "CIBW_BUILD_FRONTEND=$CIBW" >> "$GITHUB_ENV" 
+          echo "CIBW_BUILD_FRONTEND=$CIBW" >> "$GITHUB_ENV"
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@d4a2945fcc8d13f20a1b99d461b8e844d5fc6e23 # v2.21.1
+        uses: pypa/cibuildwheel@7940a4c0e76eb2030e473a5f864f291f63ee879b # v2.21.3
+        if: ${{ matrix.python[0] != 'cp313t' || runner.os != 'Windows' }}
         env:
           CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}*
           CIBW_ARCHS: ${{ matrix.buildplat[2] }}
           CIBW_PRERELEASE_PYTHONS: True
+          CIBW_FREE_THREADED_SUPPORT: ${{ endsWith(matrix.python[0], 't') }}
```

Comment:
This can simply be set to `True` unconditionally I think (like in numpy [here](https://github.com/numpy/numpy/blob/f1b2fb54b654443f1dda48641cd22da4f1f14760/.github/workflows/wheels.yml#L169)).